src/pages/Home.jsx

```diff
@@ -19,13 +19,22 @@ export default class Home extends Component {
     this.onSearchTextSubmit = this.onSearchTextSubmit.bind(this);
     this.getCategory = this.getCategory.bind(this);
     this.categoryApi = this.categoryApi.bind(this);
+    this.updateCartIcon = this.updateCartIcon.bind(this);
   }
 
   async componentDidMount() {
     const categoriesObjects = await api.getCategories();
     const categories = categoriesObjects.map(category => category);
+    this.updateCartIcon();
     this.setState({ categories });
   }
+  
+  updateCartIcon() {
+    const cart = localStorage
+    ? JSON.parse(localStorage.getItem('cart'))
+    : [];
```

Comment:
acho que aqui faltou uma indentação. eu tentei colocar tab por aqui, mas não consegui. digitei dois espaços, mas tenho medo que o lint reclame pois alguns estilos preferem que se use tab em vez de espaço

```suggestion
      const cart = localStorage
      ? JSON.parse(localStorage.getItem('cart'))
    : [];
```